### PR TITLE
create database

### PR DESCRIPTION
**Pull request by AsalaKM**
make connection
write queries for ticket

### CODE REVIEW

File: src/model/queries/ticket.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+const dbConnection = require('../dbConnection');
+
+const getLatestTicket = () => new Promise((resolve, reject) => {
```
Comment (SalwaMugh) [inline src/model/queries/ticket.js:3]: ASALA why are creating new promise while it's already available !!!

**Review by FarahZaqout — COMMENTED** [review]
Remove the test scripts to make travis pass.

**Comment by FarahZaqout** [comment]
Please resolve conflicts and relate to the proper issue in the PR body.

File: src/model/queries/ticket.js (hunk 1 above)
Comment (FarahZaqout) [reply src/model/queries/ticket.js:3]: I let them do it like this last time so they can learn how to write a promise from scratch. But as Salwa said, no need to define a new promise. dbConnection.query(sql) itself is a promise.

**Pull request closed without merging**